_posts/2016-09-30-sr2017-kickstart-dates-announced.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: news
+title: 2017 Kickstart locations announced
+---
+
+We are happy to announce the locations for our 2017 Kickstart on the 15th of
+ October 2016.
+
+Kickstart is the event which kicks off a competition year, where the game will
+be announced, and robotics kits will be handed out to the teams.
+
+The venues for Student Robotics 2017 kickstart are:
+
+- Southampton University
+- Bristol University
+- The Kings Centre, Oxford
+
+Please visit our [Kickstart page][kickstart-page] for more information. If you
+want to take part, but haven't already signed up to compete, you can join our
+waiting list by [starting a team][compete].
```

Comment:
Since this is a new website, we should decide on the "voice" we're going to use for the news articles (etc.). On the other hand, since we've imported all the old ones we should probably be consistent with them.

From a quick look we've historically chosen to use an impersonal voice:
- "The locations for Kickstart SR2017 have been announced" rather than "We are happy to announce.."
- "More information can be found on the Kickstart pgae" or "See the Kickstart page for more information." rather than "Please visit our.."

There are clearly places where a personal voice does work better though; for example I think that the "If you want to take part..." sentence here is fine.